### ISSUE TITLE
Generated bash autocomplete emits error with every tab-completion on OS X

### ISSUE THREAD

**Issue by cuongdo** [spf13/hugo#1814]
When I generate a bash autocomplete file and use it on OS X El Capitan, I get the following error when performing a tab completion:

```
-bash: declare: -A: invalid option
declare: usage: declare [-afFirtx] [-p] [name[=value] ...]
```

It looks like `declare -a` (lower-case 'a') is the right thing to use to declare an associative array on OS X's variant of bash.

**Comment by cuongdo** [comment]
Actually, this is a bug in cobra, so I'll file an issue there.

**Comment by anthonyfok** [comment]
@cuongdo has filed an issue at the Cobra GitHub page.
See spf13/cobra#240 for continued discussion.

**Comment by anthonyfok** [comment]
Hi @cuongdo,

I believe the issue is fixed at spf13/cobra#240.  Please test to see if it works for you and report back.  Many thanks!

**Comment by cuongdo** [comment]
Yes, it is fixed. Thanks!
